src/main/java/frc/robot/commands/DriveUpTheRampCommand.java

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.DrivetrainSubsystem;
+import frc.robot.subsystems.NavxSubsystem;
+
+public class DriveUpTheRampCommand extends CommandBase {
+  /** Creates a new DriveForwardCommand. */
+
+  DrivetrainSubsystem drivetrainSub;
+  NavxSubsystem navxSub;
+
+  public DriveUpTheRampCommand(DrivetrainSubsystem drivetrainSub, NavxSubsystem navxSub) {
+    // Use addRequirements() here to declare subsystem dependencies.
+    
+    this.drivetrainSub = drivetrainSub;
+    this.navxSub = navxSub;
+    addRequirements(drivetrainSub);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {}
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    if(navxSub.getPitch() > 5){
```

Comment:
An `if` statement would be appropriate in a slightly different case: If the rule were something like "if the pitch is greater than 5, drive forward, but if it is less than or equal to 5, do something different", then an `if/else` would be appropriate.  If the rule were "if the pitch is greater than 5, drive forward, but if the pitch is less than or equal to 5, don't do anything", then an `if` would be appropriate.

This is a slightly different situation: "If the pitch is less than or equal to 5, then you're done with this command."  You could even view it as "ALWAYS drive forward at 0.6, and if the pitch is less than or equal to 5, then you're done with this command."

The way you currently have it written, you're never done with this command.  `isFinished()` always returns false.